src/lib.rs

```diff
@@ -434,61 +501,61 @@ make_color! {
     ///Linear luminance.
     Luma {
         ///Linear luminance.
-        y(luma: T)[alpha: T] => new;
+        y(luma: T)[alpha: T] => with_wp;
```

Comment:
I think it's enough to change it back to use `new` for now. It won't stop anyone from using it with any other white point, by converting.